packages/grid/x-grid-modules/src/components/styled-wrappers/getStyles.ts

```diff
@@ -0,0 +1,235 @@
+import { darken, fade, lighten, makeStyles } from '@material-ui/core/styles';
+
+export const getStyles = makeStyles((theme) => ({
+  'MuiDataGrid-root': {
+    lineHeight: theme.typography.pxToRem(24),
+    'box-sizing': 'border-box',
```

Comment:
@mnajdova will we write the styles with the JavaScript Object or CSS template string synthax in v5? If with the former, 

```suggestion
    boxSizing: 'border-box',
```

would be better. As for all the other kebab case (hyphen)

Reply:
done